audit_report/3.1.0/src/00_09_introduction.rst

```diff
@@ -1,9 +1,255 @@
+   * - shake_cipher
+     - simd
+     - socket
+     - sp800_108
+     - sp800_56c
+   * - sphincsplus_common
+     - sphincsplus_sha2
+     - sphincsplus_shake
+     - stateful_rng
+     - stream
+   * - system_rng
+     - thread_utils
+     - tls
+     - tls_cbc
+     - tls12
+   * - tls13
+     - trunc_hash
+     - utils
+     - win32_stats
+     - x509
+   * - xmss
+     - xts
+     -
+     -
+     -
+
+The following previously existing modules are now in scope
+and were fully reviewed:
+
+- argon2
+- blake2
+- hkdf
+- shake
+
+
+Patch Description Content
+-------------------------
+
+Chapter 4 shows the changes for this document's review iteration for all topics in scope.
+Patches are sorted in a semantically meaningful way by assigning each one to a sensible topic.
+Each topic provides a brief description and lists the authors for the contained patches.
+Afterward, an extensive table with all related patches is provided.
+
+The table contains the pull request ids or commit hashes of the reviewed patches with a GitHub link.
+For orientation, a brief description of the patch is given. Note that
+this description is only on a summary level and does not cover all patch changes in detail. Most
+pull requests and commits are sufficiently described at the provided GitHub link.
+Also, each patch within the table is assigned a security category, and information about the approvers
+and auditors is given.
+
+
+Security Categories
+~~~~~~~~~~~~~~~~~~~
+
+For this audit, four security categories are distinguished. The category *critical* labels patches
+that apply breaking changes to cryptographic functionality, e.g., implementing a new algorithm
+or updating an old one to a new standard. Patches labeled as *relevant* are changes to cryptographic
+algorithms without breaking the algorithm's specification. Mostly, this category contains
+optimizations or major refactoring of cryptographic modules. All changes with no direct effect on
+cryptographic operations are categorized as *info*. The *out of scope* category identifies patches
+that only affect modules not in this review's scope. Patches of the last type are not reviewed
+in detail.
+
+
+Approvals and Auditors
+~~~~~~~~~~~~~~~~~~~~~~
+
+The audit process is based on two requirements. The first one is the four-eye principle, i.e., at least two individuals must vision each patch. In the best-case scenario, this is the author and a separate approver
+who performs an in-depth review. The second principle requires that at least one of the two participants
+must be involved in this audit process so that we can guarantee that the review meets the desired
+quality standard.
+
+If a patch has no participating approver, the patch is visioned in a later review by an auditor. The patch table lists this auditor in brackets, while real approvers are listed without them.
+Note that authors can edit pull requests after approval. Therefore, auditors vision additional changes after
+approval for this audit.
```

Comment:
It's admittedly quite hard to put the distinction in words. Here's my take at this:

```suggestion
The audit process is based on two requirements: 

* **The four-eye principle:** at least two individuals must inspect each patch
* **Audit quality:** at least one of the inspectors must be involved in this audit process.

Therefore, pull requests that were either authored or reviewed on GitHub by one of the members of this audit project do not require an additional in-depth review for this particular audit process.
Other patches are evaluated and reviewed by an auditor in retrospect with the result of the audit being stated in this document.
The distinction between "approvers" (of pull requests on GitHub) and "auditors" (in retrospect explicitly for this project) is visualized by setting the latter into parenthesis in the patch tables below.

Auditing members of this project and their GitHub handles are:

* Fabian Albert (@FAlbertDev)
* René Fischer (@securitykernel)
* Philippe Lieser (@lieser)
* René Meusel (@reneme)
* Amos Treiber (@atreiber94)
```

Reply:
I like your version!